src/Microsoft.VisualStudio.SDK.Analyzers.CodeFixes/VSSDK001DeriveFromAsyncPackageCodeFix.cs

```diff
@@ -212,8 +210,6 @@ private async Task<Document> ConvertToAsyncPackageAsync(CodeFixContext context,
                         return SyntaxFactory.ParenthesizedExpression(SyntaxFactory.AwaitExpression(invocation))
                             .WithAdditionalAnnotations(Simplifier.Annotation);
                     });
-
-                updatedRoot = await Utils.AddUsingTaskEqualsDirectiveAsync(updatedRoot, cancellationToken);
             }
```

Comment:
Why did we drop the call to `AddUsingTaskEqualsDirectiveAsync`? Is it just because targeting Dev17 means you don't need it? If so, we should probably condition this on targeting less than 17.0 instead of dropping it altogether. Otherwise the code fix will produce code that doesn't compile when run against 16.x extensions, I think.

Reply:
Reverted